src/components/InnerLayout/InnerLayout.module.css

```diff
@@ -0,0 +1,47 @@
+@import '_colors.css';
+@import '_fonts.css';
+@import '_sizes.css';
+
+.inner-layout {
+  display: flex;
+  box-sizing: border-box;
+  width: 100%;
+  max-width: $screenDesktopMin;
+  margin: auto;
+
+  @media (max-width: $screenTabletMax) {
+    padding: 0 1rem;
+  }
+
+  @media (max-width: $screenMobileMax) {
+    flex-direction: column;
+  }
+
+  &__left-sidebar {
+    flex: 1 1 auto;
+    box-sizing: border-box;
+    max-width: 25.625rem;
+    margin-right: 1rem;
```

Comment:
why right sidebar and left sidebar have different margins?

Reply:
It was 0:30 after an assessment day) what did you look for from me?)